lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -254,6 +261,7 @@ func (c *Config) Connection() (*ConnPool, *errors.Error) {
 	//Check if any connection error occured
 	if err != nil {
 		if errs, aye := isDbConnectError(err); aye {
+			log.Error("error while trying to get Readpool connection : " + errs.Error())
```

Comment:
please remove "error" from the log message

Reply:
will be corrected in upcoming PRs.